rpc/src/responses/balance.rs

```diff
@@ -0,0 +1,12 @@
+use super::Response;
+
+pub struct BalanceResponse {
+    pub balance: u32,
+}
+
+impl Response for BalanceResponse {
+    fn from_response_str(response: &str) -> Self {
+        let balance = response.trim().replace("\"", "").parse().unwrap();
```

Comment:
Due to the _interesting_ design choices behind responses from tezos RPC calls, the response text is not `0` but in fact `"0"\n`, so we have to trim and replace quotes for the parse to work.

Reply:
God bless Tezos overlords